src/pages/playground/Playground.tsx

```diff
@@ -723,8 +722,6 @@ const Playground: React.FC<PlaygroundProps> = ({ workspaceLocation = 'playground
 
   const onChangeMethod = React.useCallback(
     (newCode: string, delta: CodeDelta) => {
-      handleEditorValueChange(newCode);
```

Comment:
This results in the editor value being set twice, once in the `Editor` component, and once in the playground workspace. You can verify that this is safe to remove by observing that you can still change the value of the editor in the playground, or by observing that prior to this PR, the Redux action `UPDATE_EDITOR_VALUE` is fired twice for each change.